packages/nx/src/command-line/examples.ts

```diff
@@ -307,7 +307,7 @@ export const examples: Record<string, Example[]> = {
     {
       command: 'run-many --target=test --projects=*-app --exclude excluded-app',
       description:
-        'Test all projects ending with `*-app` except `excluded-app`',
+        'Test all projects ending with `*-app` except `excluded-app`.  Note: your shell may require you to escape the `*` like this: `*`',
```

Comment:
This doesn't look right. I think it may have been formatted..

You might need `\\*`?

Reply:
yep. fixed.